### DISCUSSION TITLE
v1.1.0major release not loading

### DISCUSSION THREAD

**Discussion by GeoStork** [Jyers/Marlin#88] [Q&A] [answered]
OK, have loaded firmware on my Ender many times and loaded the precompiled firmware (from 2-17-21) this morning.  Saw there was a new major release, downloaded precompiled .BIN, put that file on my "FIRMWARE" microSD (it's the only file on the card)... on reboot the machine starts up but doesn't seem to load the new firmware.  My "test" is that my offsets and extruder flow rate are not reset to a default value (they were the other times I've loaded new firmware).   Thoughts on what I"m missing?  It's been a long day.  Thanks in advance.

**Comment by Ramrawd** [comment, marked as answer]
Your defaults are stored in the EEPROM and will override the new firmware's settings. You'll want to go into the control menu and reset the stored values any time you load a new firmware. Then you can reenter your values to give you a "fresh" install. Hope that helps!

**Reply by GeoStork** [reply]
AH.  That's the trick!  LOL.  I was looking at the firmware version number and it says 1.1 not 1.1.0 so I "freaked" out.  My first pre-compilted was 5x5 and I loaded 3x3 this morning so that's why my settings were at the default, then I tuned it up and loaded tonight with newest 3x3 version and those settings were in EEPROM as you say.  Thank you!!!!

**Reply by Jyers** [reply]
Sorry for the slight confusion with 1.1 vs 1.1.0, I wasn't fully decided on the versioning format I wanted until I got the releases set up, hence the slight discrepancy in the info menu version number.
